managed/services/agents/mongodb_test.go

```diff
@@ -231,6 +231,161 @@ func TestMongodbExporterConfig226(t *testing.T) {
 	})
 }
 
+func TestMongodbExporterConfig2411(t *testing.T) {
+	pmmAgentVersion := version.MustParse("2.41.1")
```

Comment:
our dev-latest and Feature builds usually have version which looks like `2.41.1-HEAD-fasdjf`, please make sure that it works for that too, otherwise the feature will not pass QA.

Reply:
Fixed and added short test which checking this.